weatherreport.cpp

```diff
@@ -60,6 +60,7 @@ void TestRainy() {
     string report = Report(sensor);
     cout << report << endl;
     assert(report.find("rain") != string::npos);
+    assert(report.find("junk") != string::npos);
```

Comment:
Why would the use of this software expect to find `junk` in the weather report...